service-python/featuretoggle/src/lib/queues.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+from datetime import datetime
+
+import pytz
+
+from . import main, utils
+from .redis_client import RedisClient
+from .settings import queue_config, redis_config
+
+EXCHANGE = queue_config["exchange_name"]
+TOGGLE_QUEUE = queue_config["toggle_queue_name"]
+
+
+def feature_toggle_request_callback(channel, method, properties, body):
+
+    redis_client = RedisClient(redis_config)
```

Comment:
Future want-to-have: the Python Redis client already does a lot of heavy lifting (like connection pooling, etc) so it would be nice for this object to be shared amongst multiple calls instead of instantiating a new one all the time.

Reply:
Somehow I missed this the first time around; I'll add it to a ticket.